<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetre</class>
 <widget class="QWidget" name="fenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fenetre options ecran de veille</string>
  </property>
  <property name="windowIcon">
   <iconset resource="systray.qrc">
    <normaloff>:/images/preferences-desktop-screensaver-2.png</normaloff>:/images/preferences-desktop-screensaver-2.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Fichier</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ressourcePath">
       <property name="placeholderText">
        <string>Choisissez un fichier / un dossier</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Temps d'inactivité</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="timeout">
       <property name="suffix">
        <string> s</string>
       </property>
       <property name="minimum">
        <number>4</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>180</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Temps restant</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcdNBTime">
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Rafraichir général</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="timeActu">
       <property name="suffix">
        <string> msec</string>
       </property>
       <property name="minimum">
        <number>200</number>
       </property>
       <property name="maximum">
        <number>60000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Vidéos trouvées</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblVideoNb">
         <property name="whatsThis">
          <string>Nombres de vidéo compatibles dans le dossier</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="screenshot">
     <property name="title">
      <string>Activité de l'écran</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="whatsThis">
         <string>Activité si la différence entre les screnshots est importante</string>
        </property>
        <property name="text">
         <string>Différence notable sur l'écran</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="screenshotMargin">
        <property name="whatsThis">
         <string>2% est suffisant pour ne pas s'activer avec un lecteur vidéo  réduit</string>
        </property>
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Rafraichir la capture d'écran</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="screenshotTime">
        <property name="whatsThis">
         <string>compris entre 2s et 1h</string>
        </property>
        <property name="suffix">
         <string> s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mouse">
     <property name="title">
      <string>Activité de la souris</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Taux d'écart de la souris</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="mouseMargin">
        <property name="suffix">
         <string> px</string>
        </property>
        <property name="maximum">
         <number>20000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Rafraichir la souris</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="mouseTime">
        <property name="whatsThis">
         <string/>
        </property>
        <property name="suffix">
         <string> msec</string>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>3600000</number>
        </property>
        <property name="value">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mouseV">
     <property name="title">
      <string>Activité de la souris en veille</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Taux d'écart de la souris</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="mouseVMargin">
          <property name="suffix">
           <string> px</string>
          </property>
          <property name="maximum">
           <number>20000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="mouseVRepeat">
        <property name="text">
         <string>Recliquer pour ne pas perturber la frappe</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="key">
     <property name="whatsThis">
      <string>Si désactivé, le retour au bureau/menu démarrer/changer d'aplication fonctionne toujours</string>
     </property>
     <property name="title">
      <string>Action du clavier en veille</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="keyRepeat">
        <property name="text">
         <string>Renvoyer la touche pressée pour ne pas perturber la frappe</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="bActu">
       <property name="text">
        <string>Rafraichir les paramètres</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bLaunch">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Afficher</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>bLaunch</tabstop>
  <tabstop>ressourcePath</tabstop>
 </tabstops>
 <resources>
  <include location="systray.qrc"/>
 </resources>
 <connections/>
</ui>
